tests/EnumLabelTest.php

```diff
@@ -13,71 +13,31 @@ class EnumLabelTest extends TestCase
     public function test_labels_in_to_array()
     {
         $this->assertEquals([
-            'A' => 'a',
-            'B' => 'B'
-        ], EnumWithLabels::toArray());
+            'a',
+            'b'
+        ], EnumWithLabels::toLabels());
```

Comment:
to me `asLabels()` or `labels()` sounds a little more clear. Definitely not a blocker though :)

Reply:
I'm open - the `toLabels()` is just the current state - so upgrading would be easier. But true, without prefix and `to` is used by PHP core from time to time. So `asLabels()` & `asValues()` would prevent possible conflicts.
But I'm open to discussing this. I'm happy with all three solutions and each has its own pros&cons.

Reply:
I'm on the same page here :p my preference would be `as`, but you have an equal voice in this decision.

Reply:
I will tweet it out and possibly we get some community opinions on this. Just to get a feeling of the community preferred one.

Reply:
https://twitter.com/devgummibeer/status/1384831511250276352

Reply:
I would use `labels` because it's closer to the native `Enum::cases` method.

Reply:
https://github.com/spatie/enum/pull/96/commits/5c154474cbcd6cb9fd650b150b0078b2614535d4